.eslintignore

```diff
@@ -0,0 +1 @@
+.eslintigrore
```

Comment:
Why don't you put back `.eslintrc.js` as it was and we can try and run through the issues together. And in this file, restore the line as follow.

```suggestion
.eslintrc.js
```